src/EMG.Lambda.LocalRunner/Internal/InnerReceivingRunnerBuilder.cs

```diff
@@ -14,14 +14,15 @@ public class InnerReceivingRunnerBuilder<TInput> : IReceivingRunnerBuilder<TInpu
         public int Port { get; set; }
 
         public Func<ILambdaSerializer> SerializerFactory { get; set; }
-
+        public string ResponseContentType { get; internal set; }
```

Comment:
Not a great fan of `internal set`. I like to hide members only via interface declaration, not via accessor.

e.g.
```csharp
public interface ISomething
{
    string Something { get; }
}

public class MySomething : ISomething
{
    public string Something { get; set; }
}

ISomething thing1 = new MySomething();
thing1.Something = "asd"; // compiler not happy

var thing2 = new MySomething();
thing2.Something = "foo"; // compiler happy!
```

Reply:
Just a VS Code/Omnisharp default that I missed during pre-commit review, I'll fix it :smile: